pkg/build/Godeps/Godeps.json

```diff
@@ -3,10 +3,18 @@
 	"GoVersion": "go1.7",
 	"GodepVersion": "v74",
 	"Packages": [
-		"github.com/fsouza/go-dockerclient/..."
+		"github.com/fsouza/go-dockerclient"
 	],
 	"Deps": [
 		{
+			"ImportPath": "github.com/Azure/go-ansiterm",
+			"Rev": "70b2c90b260171e829f1ebd7c17f600c11858dbe"
+		},
+		{
+			"ImportPath": "github.com/Azure/go-ansiterm/winterm",
+			"Rev": "70b2c90b260171e829f1ebd7c17f600c11858dbe"
+		},
```

Comment:
why isn't it sufficient that origin itself already godeps this?  are the godeps not unioned together?